### PR TITLE
Convert more tests to use junit5

### PR DESCRIPTION
**Pull request by nielsm5**

### CODE REVIEW

**Review by gvanbrakel — APPROVED** [review]

File: akamai/src/test/java/nl/nn/adapterframework/extensions/akamai/NetStorageCmsSignerTest.java [hunk 1]
```diff
@@ -22,7 +22,7 @@
 import java.util.Map;
 
 import org.apache.commons.codec.binary.Base64;
-import org.junit.Test;
+import org.junit.jupiter.api.Test;
```
Comment (tnleeuw) [inline akamai/src/test/java/nl/nn/adapterframework/extensions/akamai/NetStorageCmsSignerTest.java:25]: Why not also import JUnit5 assertions?

File: core/src/test/java/nl/nn/adapterframework/xml/ClassLoaderURIResolverTest.java [hunk 2]
```diff
@@ -7,19 +7,17 @@
...
 import javax.xml.transform.Source;
 import javax.xml.transform.TransformerException;
 
 import org.apache.logging.log4j.Logger;
-import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.junit.runners.Parameterized;
-import org.junit.runners.Parameterized.Parameter;
-import org.junit.runners.Parameterized.Parameters;
+import org.junit.jupiter.params.ParameterizedTest;
```
Comment (tnleeuw) [inline core/src/test/java/nl/nn/adapterframework/xml/ClassLoaderURIResolverTest.java:18]: Import JUnit5 Assertions too, perhaps?

File: core/src/test/java/nl/nn/adapterframework/xml/ClassLoaderURIResolverTest2.java [hunk 3]
```diff
@@ -10,9 +10,9 @@
 
 import javax.xml.transform.TransformerException;
 
-import org.apache.xerces.xni.XNIException;
-import org.junit.Ignore;
-import org.junit.Test;
+import org.junit.jupiter.api.Test;
```
Comment (tnleeuw) [inline core/src/test/java/nl/nn/adapterframework/xml/ClassLoaderURIResolverTest2.java:13]: Importing JUnit5 Assertions too?

File: core/src/test/java/nl/nn/adapterframework/xml/SaxExceptionTest.java [hunk 4]
```diff
@@ -4,8 +4,8 @@
 import static org.hamcrest.core.IsNot.not;
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertNotNull;
+import static org.junit.jupiter.api.Assertions.fail;
```
Comment (tnleeuw) [inline core/src/test/java/nl/nn/adapterframework/xml/SaxExceptionTest.java:7]: Import other assertions from JUnit5 package too, perhaps?

**Review by tnleeuw — CHANGES_REQUESTED** [review]

**Review by tnleeuw — APPROVED** [review]

**Pull request merged**